unitTest/@testIGAMembraneAnalysis/testNonlinearMembraneAnalysis.m

```diff
@@ -38,7 +38,7 @@ function testNonlinearMembraneAnalysis(testCase)
 %% 0. Read input
 
 % Define absolute tolerance
-absTol = 1e-15;
+absTol = 1e-12;
```

Comment:
Please revert the changes in the unit tests, I would like to fix them myself under Windows.